app/src/main/java/com/example/nvdovin/weatherapp/presentation/history/grid/adapter/GridHistoryAdapter.java

```diff
@@ -0,0 +1,81 @@
+package com.example.nvdovin.weatherapp.presentation.history.grid.adapter;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import com.example.nvdovin.weatherapp.R;
+import com.example.nvdovin.weatherapp.data.model.WeatherData;
+import com.example.nvdovin.weatherapp.domain.utils.time.TimeUtils;
+import com.example.nvdovin.weatherapp.domain.utils.design.ImageUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GridHistoryAdapter extends BaseAdapter {
+
+    private static final String HOUR_PRECISION = ":00";
```

Comment:
hour_precission? really? so, j could you explain please why do you named it in this manner?